Traceback (most recent call last):
  File "C:\Users\janko\AppData\Local\Programs\Python\Python310\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\janko\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\janko\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\janko\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\janko\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 646, in run_until_complete
    return future.result()
  File "C:\Users\janko\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\janko\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\janko\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
labview = open('zapis_labview.lvm', mode='r') # Odpremo datoteko

seznam = []  # Pripravimo prazen seznam, na katerega bomo pripenjali vrednosti  
for vrstica in labview:
    podatek = vrstica.split()  # Vrstico razbijemo na dva stolpca
    if len(podatek) is 2:
         seznam.append((podatek[0], podatek[1]))  # V seznam shranjujemo dvojico terk, bomo rabili za narediti slovar
    else:
        break  # Ko pride do End of header se ustavi
        
labview.close() # Zapremo datoteko

# Naredimo slovar
slovar = dict(seznam)

print(slovar)

# Preverimo ƒçe dela
print(slovar['Time'])

------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mFileNotFoundError[0m                         Traceback (most recent call last)
Input [1;32mIn [5][0m, in [0;36m<cell line: 1>[1;34m()[0m
[1;32m----> 1[0m labview [38;5;241m=[39m [38;5;28;43mopen[39;49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43mzapis_labview.lvm[39;49m[38;5;124;43m'[39;49m[43m,[49m[43m [49m[43mmode[49m[38;5;241;43m=[39;49m[38;5;124;43m'[39;49m[38;5;124;43mr[39;49m[38;5;124;43m'[39;49m[43m)[49m [38;5;66;03m# Odpremo datoteko[39;00m
[0;32m      3[0m seznam [38;5;241m=[39m []  [38;5;66;03m# Pripravimo prazen seznam, na katerega bomo pripenjali vrednosti  [39;00m
[0;32m      4[0m [38;5;28;01mfor[39;00m vrstica [38;5;129;01min[39;00m labview:

[1;31mFileNotFoundError[0m: [Errno 2] No such file or directory: 'zapis_labview.lvm'
FileNotFoundError: [Errno 2] No such file or directory: 'zapis_labview.lvm'

